Services/AzineHybridAuth.php

```diff
@@ -39,6 +39,11 @@ class AzineHybridAuth {
 	private $storeAsCookie;
 
 	/**
+	 * @var int
+	 */
+	private $expiresInDays;
```

Comment:
$expiresInDays is never initialized. I guess it should be part of the __constructor.